src/index.js

```diff
@@ -67,6 +68,9 @@ class ReactToPrint extends React.Component {
       return;
     }
 
+    /** clear prev print window in here */
+    this.currentPrintWindow && this.removeWindow(this.currentPrintWindow);
```

Comment:
How does this work? `currentPrintWindow` is never assigned to anything, so this will never actually remove the previous print window